addons/mail/static/src/components/chat_window_manager/chat_window_manager_tests.js

```diff
@@ -975,37 +947,24 @@ QUnit.test('[technical] chat window: composer state conservation on toggle home
         "composer should have 2 total attachments after adding 2 attachments"
     );
 
-    await this.hideHomeMenu();
-    assert.strictEqual(
-        document.querySelector(`.o_ComposerTextInput_textarea`).value,
-        "XDU for the win !",
-        "Chat window composer should still have the same input after hiding home menu"
-    );
-    assert.containsN(
-        document.body,
-        '.o_Composer .o_Attachment',
-        2,
-        "Chat window composer should have 2 attachments after hiding home menu"
-    );
+    await afterNextRender(() => this.env.messaging.discuss.update({ isOpen: true }));
+    assert.containsNone(document.body, '.o_ChatWindow', "should not have any chat window after showing discuss");
 
-    // Show home menu
-    await this.showHomeMenu();
+    await afterNextRender(() => this.env.messaging.discuss.update({ isOpen: false }));
     assert.strictEqual(
         document.querySelector(`.o_ComposerTextInput_textarea`).value,
         "XDU for the win !",
-        "chat window composer should still have the same input showing home menu"
+        "chat window composer should still have the same input after hiding discuss"
```

Comment:
Would prefer wording open/close rather than show/hide for the discuss state. This applies to all changes in this file

Reply:
done